Knowledgebase/blogs.csv

```diff
@@ -80,3 +80,4 @@ https://vld-nav.com/,class,t-body
 https://aardvarklabs.blog/,class,entry-content
 https://blog.wingate365.com/,class,post
 https://manishkutar.wordpress.com/,class,content-area
+https://fbakkensen.github.io,class,entry-content
```

Comment:
Hi, sorry for long response, conference season. 
I checked your website and the class should be `page-content` instead of `entry-content`. 
Please fix and I'll add it immidiately.